.gitlab/deps_build.yml

```diff
@@ -31,16 +31,12 @@
       -DLLVM_PARALLEL_LINK_JOBS=4 \
       ../llvm
     - cmake --build . --target install
-    - cd $DATADOG_AGENT_EMBEDDED_PATH
-    - rm -rf bin share libexec lib/clang lib/cmake lib/*.so*
-    - mkdir -p $CI_PROJECT_DIR/.tmp
-    - tar cvaf $CI_PROJECT_DIR/.tmp/clang-$ARCH-11.0.1.tar.xz .
-    - $S3_CP_CMD $CI_PROJECT_DIR/.tmp/clang-$ARCH-11.0.1.tar.xz $S3_PERMANENT_ARTIFACTS_URI/clang-$ARCH-11.0.1.tar.xz
+    - $S3_CP_CMD $DATADOG_AGENT_EMBEDDED_PATH/bin/clang $S3_PERMANENT_ARTIFACTS_URI/clang.${ARCH} 
+    - $S3_CP_CMD $DATADOG_AGENT_EMBEDDED_PATH/bin/clang-11 $S3_PERMANENT_ARTIFACTS_URI/clang-11.${ARCH} 
+    - $S3_CP_CMD $DATADOG_AGENT_EMBEDDED_PATH/bin/llc $S3_PERMANENT_ARTIFACTS_URI/llc.${ARCH}
   timeout: 2h 00m
   artifacts:
     expire_in: 2 weeks
-    paths:
-      - $CI_PROJECT_DIR/.tmp/clang-$ARCH-11.0.1.tar.xz
```

Comment:
It would be nice to include the two binaries here, since it makes it easy to download them for use in our agent VM.